meowlflow/integrations/sentry.py

```diff
@@ -0,0 +1,27 @@
+import click
+import sentry_sdk as sentry_sdk
```

Comment:
importing with `as` to explicitly re-export the package

Reply:
wat!!!